charts/linkerd/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "v1"
+appVersion: stable-2.4.0
+name: "linkerd"
```

Comment:
There already is a `linkerd` chart for linkerd 1, so we should name this one `linkerd2`, which also implies renaming the dir.

Reply:
just curious: do we know if / how much the linkerd1 chart is used? we were in a similar situation with brew, and opted to just overwrite the existing `brew install linkerd` with the latest version.

Reply:
Per chat with @grampelberg, we will rename the chart to `linkerd2`.